2021/02/23 17:03:48 Ass_main.py[line:69] INFO ######## Tools start ########
2021/02/23 17:03:48 Ass_main.py[line:165] INFO ######>>> Check device
2021/02/23 17:03:48 Ass_main.py[line:180] DEBUG Device list:[]
2021/02/23 17:03:48 Ass_main.py[line:430] INFO ######>>> Check port
2021/02/23 17:03:48 Ass_main.py[line:438] DEBUG Port list:{}
2021/02/23 17:03:48 Ass_main.py[line:966] INFO ######>>> Init settings AT Dialog
2021/02/23 17:03:48 Ass_main.py[line:1030] INFO ######>>> Init Default settings Dialog
2021/02/23 17:03:50 Ass_main.py[line:430] INFO ######>>> Check port
2021/02/23 17:03:50 Ass_main.py[line:438] DEBUG Port list:{}
2021/02/23 17:03:50 Ass_main.py[line:430] INFO ######>>> Check port
2021/02/23 17:03:50 Ass_main.py[line:438] DEBUG Port list:{}
2021/02/23 17:03:50 Ass_main.py[line:430] INFO ######>>> Check port
2021/02/23 17:03:50 Ass_main.py[line:438] DEBUG Port list:{}
2021/02/23 17:03:53 Ass_main.py[line:444] INFO ######>>> Open port
2021/02/23 17:03:53 Ass_main.py[line:447] INFO Port:  baudrate:115200  bytesize:8  parity:N  stopbits:1
2021/02/23 17:03:55 Ass_main.py[line:463] INFO ######>>> Close port
2021/02/23 17:03:55 Ass_main.py[line:463] INFO ######>>> Close port
2021/02/23 17:03:55 Ass_main.py[line:463] INFO ######>>> Close port
2021/02/23 17:03:55 Ass_main.py[line:463] INFO ######>>> Close port
2021/02/23 17:03:55 Ass_main.py[line:463] INFO ######>>> Close port
2021/02/23 17:03:55 Ass_main.py[line:463] INFO ######>>> Close port
2021/02/23 17:03:56 Ass_main.py[line:463] INFO ######>>> Close port
2021/02/23 17:03:56 Ass_main.py[line:463] INFO ######>>> Close port
2021/02/23 17:03:56 Ass_main.py[line:463] INFO ######>>> Close port
2021/02/23 17:04:05 Ass_main.py[line:444] INFO ######>>> Open port
2021/02/23 17:04:05 Ass_main.py[line:447] INFO Port:  baudrate:115200  bytesize:8  parity:N  stopbits:1
2021/02/23 17:04:07 Ass_main.py[line:597] WARNING Attempting to use a port that is not open
2021/02/23 17:04:07 Ass_main.py[line:588] WARNING 终端状态异常
2021/02/23 17:04:07 Ass_main.py[line:597] WARNING Attempting to use a port that is not open
2021/02/23 17:04:08 Ass_main.py[line:597] WARNING Attempting to use a port that is not open
2021/02/23 17:04:08 Ass_main.py[line:597] WARNING Attempting to use a port that is not open
2021/02/23 17:04:09 Ass_main.py[line:597] WARNING Attempting to use a port that is not open
2021/02/23 17:04:09 Ass_main.py[line:597] WARNING Attempting to use a port that is not open
2021/02/23 17:04:10 Ass_main.py[line:597] WARNING Attempting to use a port that is not open
2021/02/23 17:04:10 Ass_main.py[line:597] WARNING Attempting to use a port that is not open
2021/02/23 17:04:10 Ass_main.py[line:597] WARNING Attempting to use a port that is not open
2021/02/23 17:04:10 Ass_main.py[line:597] WARNING Attempting to use a port that is not open
2021/02/23 17:04:10 Ass_main.py[line:597] WARNING Attempting to use a port that is not open
2021/02/23 17:04:10 Ass_main.py[line:597] WARNING Attempting to use a port that is not open
2021/02/23 17:04:11 Ass_main.py[line:597] WARNING Attempting to use a port that is not open
2021/02/23 17:04:11 Ass_main.py[line:597] WARNING Attempting to use a port that is not open
2021/02/23 17:04:11 Ass_main.py[line:118] WARNING force stop test !!!
2021/02/23 17:04:11 Ass_main.py[line:118] WARNING force stop test !!!
2021/02/23 17:04:12 Ass_main.py[line:118] WARNING force stop test !!!
2021/02/23 17:04:12 Ass_main.py[line:118] WARNING force stop test !!!
2021/02/23 17:04:12 Ass_main.py[line:118] WARNING force stop test !!!
2021/02/23 17:04:12 Ass_main.py[line:118] WARNING force stop test !!!
2021/02/23 17:04:12 Ass_main.py[line:118] WARNING force stop test !!!
2021/02/23 17:04:15 Ass_main.py[line:118] WARNING force stop test !!!
2021/02/23 17:04:15 Ass_main.py[line:118] WARNING force stop test !!!
2021/02/23 17:04:15 Ass_main.py[line:118] WARNING force stop test !!!
2021/02/23 17:04:15 Ass_main.py[line:118] WARNING force stop test !!!
2021/02/23 17:04:17 Ass_main.py[line:597] WARNING Attempting to use a port that is not open
2021/02/23 17:04:17 Ass_main.py[line:588] WARNING 终端状态异常
2021/02/23 17:04:17 Ass_main.py[line:597] WARNING Attempting to use a port that is not open
2021/02/23 17:04:18 Ass_main.py[line:597] WARNING Attempting to use a port that is not open
2021/02/23 17:04:20 Ass_main.py[line:597] WARNING Attempting to use a port that is not open
2021/02/23 17:04:20 Ass_main.py[line:597] WARNING Attempting to use a port that is not open
2021/02/23 17:04:20 Ass_main.py[line:597] WARNING Attempting to use a port that is not open
2021/02/23 17:04:20 Ass_main.py[line:597] WARNING Attempting to use a port that is not open
